classes/class-uagb-front-assets.php

```diff
@@ -58,8 +58,53 @@ public static function get_instance() {
 	public function __construct() {
 		add_action( 'wp', array( $this, 'set_initial_variables' ), 99 );
 		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_asset_files' ) );
+
+		if ( is_multisite() && ! is_super_admin() && ! current_user_can( 'manage_network' ) ) {
+			add_filter( 'render_block', array( $this, 'render_icons_dynamically' ), 10, 2 );
+		}
+		
+	}
+
+	/**
+	 * Render icons dynamically for multisite.
+	 *
+	 * @param string $block_content the block data.
+	 * @since x.x.x
+	 * @return string|void
```

Comment:
```suggestion
	 * @return string
```
Just check once, we are not returning void

Reply:
Updated